src/engine/v3/V3Interpreter.v3

```diff
@@ -1493,7 +1493,10 @@ class V3Interpreter extends WasmStack {
 		}
 	}
 	// Returns whether the suspend got handled.
-	def tryHandleSuspend(cont: Continuation, tag: Tag, vals: Array<Value>) -> bool {
+	def tryHandleSuspend(cont: Continuation, tag: Tag, vals: Array<Value>, is_switch: bool) -> bool {
```

Comment:
If we add a new case, then we could also split this to a different function and factor out some commonalities between the two. I think that's cleaner than a `bool` that changes behavior.